### PR TITLE
fix: pass `light` prop into dropdown

### LINKED ISSUE #3609: Listbox components not receiving `light` prop
When passing the `light` prop into React listbox components, the prop is not passed down into the underlying `<Listbox>`

from the React dropdown story:

image

the `bx--list-box--light` class should also be applied when the `light` knob is `true`

### PR DESCRIPTION
**Pull request by emyarod**
Closes #3609

This PR passes the `light` prop into the underlying `<ListBox>` for the React dropdown

refs https://github.com/carbon-design-system/design-language-website/issues/135

#### Testing / Reviewing

Ensure the correct classes are applied to the dropdown's listbox when the `light` prop is true

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
LGTM 👍

**Comment by tw15egan** [comment]
More of a design question, but when the light class is active, should the menu also be styled white? Or is it supposed to strictly style the input? Asking because, in this PR (https://github.com/carbon-design-system/carbon/pull/3595), we style the dropdown menu to be light, as well as the dropdown itself. 

Can be a separate PR to update the styles there (if needed)

File: packages/react/src/components/ComboBox/ComboBox.js [hunk 1]
```diff
@@ -292,6 +292,7 @@ export default class ComboBox extends React.Component {
           clearSelection,
         }) => (
           <ListBox
+            light
```
Comment (asudoh) [inline packages/react/src/components/ComboBox/ComboBox.js]: Seems that you are trying to pass along `light` prop to `<ListBox>`, but the syntax here seems to just hard-code `light`. Probably you meant `light={light}`?

Comment (emyarod) [reply packages/react/src/components/ComboBox/ComboBox.js]: yeah good catch, and it's already passed into the listbox for all components except dropdown!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @emyarod for the update!

**Pull request merged**